plugin/cypress.config.ts

```diff
@@ -18,13 +18,14 @@ export default defineConfig({
   videosFolder: 'cypress/videos',
 
   env: {
-    OC_CLUSTER_USER: 'jenkins', // default value for jenkins
-    OC_IDP: 'my_htpasswd_provider', // default value for jenkins, can vary based on cluster setup
     'cypress-react-selector': {
       root: '#root'
     },
+    OSSMC: true,
```

Comment:
only nitpick, but this is why I would prefer to have two different configs, so there is no risk for them to be overwritten when copying the code

Reply:
Actually there are two configs, the `cypress.config` from Kiali repo and `cypress.config` from OSSMC repo. What I don't see is to have two configs just for OSSMC.